packages/expo/src/MailComposer/ExponentMailComposer.web.ts

```diff
@@ -0,0 +1,33 @@
+import array from 'cast-array';
+import filter from 'object-filter';
+import qs from 'query-string';
+
+export default {
+  get name() {
+    return 'ExponentMailComposer';
+  },
+  async composeAsync(options) {
+    const check = value => (value ? array(value).join(',') : undefined);
+
+    const to = check(options.recipients);
+    let email = filter(
+      {
+        to,
+        cc: check(options.ccRecipients),
+        bcc: check(options.bccRecipients),
+        subject: options.subject,
+        body: options.body,
+      },
+      Boolean
+    );
+
+    delete email.to;
+
+    const query = qs.stringify(email);
+    const queryComponent = query ? '?' + query : '';
+    const recipientComponent = to || '';
+    const mailto = `mailto:${recipientComponent}${queryComponent}`;
+
+    global.open(mailto);
```

Comment:
```suggestion
    window.open(mailto);
```

`open` is a method on the HTMLWindow object, not a globally defined function. It happens that window == global but the intent here is to operate on the Window object. https://html.spec.whatwg.org/multipage/window-object.html#the-window-object:dom-open